src/goSuggest.ts

```diff
@@ -97,7 +97,7 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider,
 		}
 
 		return runGodoc(path.dirname(item.fileName), item.package || path.dirname(item.fileName), item.receiver, item.label, token).then(doc => {
-			item.documentation = new vscode.MarkdownString(doc);
+			item.documentation = new vscode.MarkdownString(doc.trimLeft());
```

Comment:
Same question here, can we have all the doc trim fixes as part of a separate fixes.
Helps a lot with git blame in the future